src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -350,11 +460,42 @@ limitations under the License.
       <arg name="ModeToReturn" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
     </command>
     <command source="client" code="0x03" name="ClearWeeklySchedule" optional="true">
-      <description>
-        The Clear Weekly Schedule command is used to clear the weekly schedule.
-      </description>
+      <description>This command is used to clear the weekly schedule. The Clear weekly schedule has no payload.</description>
       <access op="invoke" privilege="manage"/>
+    </command>  
+    <command source="client" code="0x06" name="SetActivePresetRequest" optional="true">
+      <description>Upon receipt, this SHALL set the thermostat</description>
```

Comment:
This description, and all the other new descriptions, makes no sense becaues it seems cut off before it says anything useful.

Reply:
hmmm....yeah.  Previous rounds of generation had much better descriptions so I didn't bother to look at them.  I'll manually fix them up.  Thanks.